packages/it-to-browser-readablestream/index.js

```diff
@@ -1,9 +1,19 @@
 'use strict'
 
+/** @type {typeof window} */
+// @ts-ignore
 const globalThis = require('@ungap/global-this')
 
+/**
+ * @template T
+ * @param {AsyncIterator<T>|Iterator<T>} source
+ * @param {QueuingStrategy<T>} [queuingStrategy]
+ * @returns {ReadableStream<T>}
+ */
 function itToBrowserReadableStream (source, queuingStrategy = {}) {
-  return new globalThis.ReadableStream({
+  /** @type {UnderlyingSource<T> & { _cancelled:boolean} } */
+  const pump = {
+    _cancelled: false,
     async start () {
```

Comment:
From what I understand given [coveralls report](https://coveralls.io/builds/32423031/source?filename=packages%2Fit-to-browser-readablestream%2Findex.js#L17) it seems to conclude that coverage has decreased because `start` here is never called. Seems like a wrong conclusion, because all this change does is moved `ReadableStream` parameter to `pump` variable so it's easier to type annotate.

I can awkward inline type annotation, or suppress TS error (which otherwise complains about unknown `_cancelled` field). But it seems like ignoring coverall here might be a best option.

Reply:
I changed it to do inline thing to workaround coveralls.